dirsrvtests/tests/suites/config/compact_test.py

```diff
@@ -72,6 +76,64 @@ def test_compaction_interval_and_time(topo):
     inst.deleteErrorLogs(restart=False)
 
 
+@pytest.mark.ds4778
+@pytest.mark.bz1748441
+@pytest.mark.skipif(ds_is_older("1.4.3.23"), reason="Not implemented")
+def test_no_compaction(topo):
```

Comment:
Something not clear to me. compaction occurs on checkpointing (via checkpointing thread dynamic) and on compact task.
When sleeping 3 seconds are we waiting for the checkpointing to occur ? But if there is no update, is the checkpointing still trying to compact ?
I would rather expect to trigger the compact via the task, then check the message in the error log. But here I see no task.

Reply:
@tbordaz 
I took the test_compaction_interval_and_time from Mark which sets nsslapd-db-compactdb-interval and nsslapd-db-compactdb-time. Then it waits and checks logs that compaction occurred. 
What I did in my test was that I set nsslapd-db-compactdb-interval to 0, which should not trigger any compaction and then I check the logs to confirm there was none. I just decreased the time.sleep as Mark suggested.
So I would say yes, we are waiting for the checkpointing if I understand correctly.

Reply:
@bsimonova , sorry my fault. Checkpointing thread runs periodically (2.5sec) and if nsslapd-db-compactdb-interval is not null it calls a bdb_compact that logs "Compacting databases" and calls database compaction. So even if database compaction does nothing (no updates), the log is present.
Sorry for the noise